mapswipe_workers/tests/unittests/fixtures/build_area/projects.csv

```diff
@@ -0,0 +1 @@
+2020-02-27 13:13:11.41795	test	0106000020E6100000010000000103000000010000000500000005000020E87C4140DF0CEE9456CC2FC0030000406E8B4140DF0CEE9456CC2FC0030000406E8B4140743F7755D7A32FC005000020E87C4140743F7755D7A32FC005000020E87C4140DF0CEE9456CC2FC0		f	buildings	test - Malawi (1)\ntest	0	test	test_build_area	1	15120	0	inactive	3	{"groupMaxSize": 0, "projectTopic": null, "projectRegion": null, "projectNumber": null, "requestingOrganisation": null, "groupSize": 120, "zoomLevel": 18, "tileServer": {"name": "bing", "url": "https://ecn.t0.tiles.virtualearth.net/tiles/a{quad_key}.jpeg?g=7505&mkt=en-US&token={key}", "apiKeyRequired": null, "apiKey": "AopsdXjtTu-IwNoCTiZBtgRJ1g7yPkzAi65nXplc-eLJwZHYlAIf2yuSY_Kjg3Wn", "wmtsLayerName": null, "captions": null, "date": null, "credits": "\\u00a9 2019 Microsoft Corporation, Earthstar Geographics SIO"}}
```

Comment:
should this file also contain the other projects listed in projects_drafts? Or are you going to test it only for a single project?